asv_bench/benchmarks/hdfstore_bench.py

```diff
@@ -40,13 +48,7 @@ def setup(self):
 
     def teardown(self):
         self.store.close()
-        self.remove(self.f)
-
-    def remove(self, f):
-        try:
-            os.remove(f)
-        except:
-            pass
+        os.remove(self.f)
```

Comment:
it's not needed to keep the try .. except here?

Reply:
(and I seem to remember another PR recently related to this 'remove' ?)

Reply:
@jorisvandenbossche : Yes, you are correct, as that was me who made the comment 😉 .  @mroeschke : revert this change to restore the `try-except`.  Windows is notorious for surprising users on `os.remove`.  No reason to crash benchmarks for something like that.

Reply:
@jorisvandenbossche : What do you think about just moving that function `pandas_vb_common` ?  It's not really an instance method but rather a utility from which a lot of classes could benefit.

Reply:
Or put it in a `Benchmark` base class to inherit from, both are ok to me (if we do the base class, we could also put the `goal_time = 0.2` there instead of repeating it everywhere

Reply:
I'm probably -0 on abstracting `goal_time` into a base class.  It might be a little nicer but adds some indirection to the benchmarks that I'm uncertain about.

Reply:
What do you mean with "adds some indirection to the benchmarks" ?

Reply:
BTW, it is what scipy does: https://github.com/scipy/scipy/blob/master/benchmarks/benchmarks/common.py#L13

Reply:
note we *already* have a base class for this ``_BenchTeardown`` (PR was pretty recent) in ``io_bench.py``. So I would simply move this *into* a top-level class you can inherit from.

Reply:
> What do you mean with "adds some indirection to the benchmarks" ?

Just that it wouldn't be immediately clear what the goal time for the benchmarks would be.